docs/model_developers/handlers.rst

```diff
@@ -95,6 +95,17 @@ Each Handler should override 4 methods of the CRUDHandler:
 The context (See :class:`~inmanta.agent.handler.HandlerContext`) passed to most methods is used to
 report results, changes and logs to the handler and the server.
 
+The :meth:`~inmanta.agent.handler.HandlerContext.set_fact` method can be used to report facts to the server.
```

Comment:
I would also link to std::get_fact

and link from the glossary to here.

And I would explain what facts are in a few words.